CODEOWNERS

```diff
@@ -3,6 +3,4 @@
 # PRs created will not automatically request reviews from the team.
 #
 # Teams:
-
-# Individuals:
-* @glitchcrab
+* @giantswarm/team-halo-engineers
```

Comment:
@glitchcrab please cry if you're not okay here

Reply:
![sadpanda](https://user-images.githubusercontent.com/7832341/127304170-9e266fb4-409a-423f-920b-65c655862106.png)